tests/e2e/tests/e2e/GitSsh.spec.ts

```diff
@@ -103,15 +112,16 @@ suite('Git with ssh workflow', async () => {
 
         data.metadata!.name = wsNameCheckPropagatingKeys;
         await testWorkspaceUtils.createWsFromDevFile(data);
-        // update view of dashboard, it's related to https://github.com/eclipse/che/issues/19020
-        await driverHelper.reloadPage();
-        await dashboard.waitPage();
-        await driverHelper.navigateToUrl(workspacePrefixUrl + wsNameCheckPropagatingKeys);
+        await dashboard.openDashboard();
+        await browserTabsUtil.navigateTo(workspacePrefixUrl + wsNameCheckPropagatingKeys);
         await ide.waitWorkspaceAndIde();
         await projectTree.openProjectTreeContainer();
         await driverHelper.wait(TimeoutConstants.TS_SELENIUM_LOAD_PAGE_TIMEOUT);
         await cloneTestRepo();
-        await projectTree.waitItem('Spoon-Knife');
+        // workaround for issue: https://github.com/eclipse/che/issues/19101
```

Comment:
the actual issue for tracking now is https://github.com/eclipse/che/issues/19544

Reply:
Thanks, replaced.